quantum/process_keycode/process_unicode_common.c

```diff
@@ -120,3 +123,27 @@ void register_hex(uint16_t hex) {
     unregister_code(hex_to_keycode(digit));
   }
 }
+
+void send_unicode_hex_string(const char *str)
+{
+  if (!str) { return; } // Saftey net
+  while (*str) {
+    // Find the next code point (token) in the string
+    for (; *str == ' '; str++);
+    size_t n = strcspn(str, " "); // Length of the current token
+    char code_point[n+1];
+    strncpy(code_point, str, n);
+    code_point[n] = '\0'; // Make sure it's null-terminated
+
+    // Normalize the code point: make all hex digits lowercase
+    for (char *p = code_point; *p; p++) {
+      *p = tolower(*p);
```

Comment:
Cast `*p` in the function call to `unsigned char` to avoid the warning that breaks CI. [Details](https://stackoverflow.com/a/23507563/1523774)

```c
*p = tolower((unsigned char)*p);
```